packages/mobile/locales/es-419/accountScreen10.json

```diff
@@ -4,21 +4,21 @@
   "languageSettings": "Idioma",
   "localCurrencySetting": "Moneda",
   "licenses": "Licencias",
-  "removeAccountTitle": "Reiniciar Valora",
-  "removeAccountDetails": "Reiniciar borrará tu cuenta de este dispositivo. Tu cuenta mantendrá el dinero que tenga, pero solo será recuperable con tu Clave de Cuenta",
+  "removeAccountTitle": "Reinicializar Valora",
+  "removeAccountDetails": "Al reiniciar se borrará tu cuenta de este dispositivo. Tu cuenta mantendrá los fondos que tenga, pero solo la podrás recuperar con tu Clave de Cuenta.",
   "legal": "Legal",
   "securityAndData": "Seguridad y datos",
-  "requirePinOnAppOpen": "Requerir PIN en la aplicación abierta",
+  "requirePinOnAppOpen": "Requerir PIN cuando al app esté abierta",
```

Comment:
```suggestion
  "requirePinOnAppOpen": "Requerir PIN cuando la aplicación esté abierta",
```

Reply:
I saw uses of both *app* and *aplicación* throughout the strings so I am not sure if we want to stick to one. 

If we do want to keep `app` for this particular case then it should be either `el app` or `la app` instead of the current`al app`

Reply:
I think there was a preference for using `app` vs `aplicación` -- will update to `la app`